cmake/external/firestore.cmake

```diff
@@ -18,12 +18,12 @@ if(TARGET firestore)
   return()
 endif()
 
-set(version 9bae3613d885fb8cc7d74c612c90f821a0e1981f)
 ExternalProject_Add(
   firestore
 
   DOWNLOAD_DIR ${FIREBASE_DOWNLOAD_DIR}
-  URL https://github.com/firebase/firebase-ios-sdk/archive/${version}.tar.gz
+  GIT_REPOSITORY https://github.com/firebase/firebase-ios-sdk
```

Comment:
If we make this change, we should probably also add `GIT_TAG` to pin it to a specific commit. As it is I'm guessing it will just get HEAD from the master branch.

Reply:
Nevermind my comment. I didn't see the entire diff and you *did* specify `GIT_TAG`.